lib/sproxyd.js

```diff
@@ -12,6 +12,11 @@ const keygen = require('./keygen');
  */
 function _createRequest(req, log, callback) {
     const request = http.request(req, response => {
+        // Consume the response body first when not relevant for the
+        // request type
+        if (req.method === 'PUT' || req.method === 'DELETE') {
```

Comment:
Question, should "post" be part of the list ? (I am thinking of batch delete)

Reply:
Good point, you may be right, I will have to check this (tomorrow)

Reply:
I fixed it